lib/zlib.js

```diff
@@ -350,8 +350,7 @@ Object.defineProperty(Zlib.prototype, 'bytesRead', {
 // `params()` function should not happen while a write is currently in progress
 // on the threadpool.
 function paramsAfterFlushCallback(level, strategy, callback) {
-  if (!this._handle)
```

Comment:
I think these were all optimization to avoid an unnecessary function call in the common case.

Reply:
That should not be a significant overhead. Such a micro optimization should not have any impact on a real application.